src/components/targets/CreateTargetForm.js

```diff
@@ -77,21 +91,27 @@ const CreateTargetForm = ({ intl, newTargetlatlng, setNewTargetlatlng }) => {
 
   return (
     <form className="create-target-form" onSubmit={handleSubmit}>
-      <FormInput
-        labelClassName="create-target-form__text"
-        inputClassName="create-target-form__input"
-        inputLabel={intl.formatMessage({
+      <p className="create-target-form__text">
+        {intl.formatMessage({
           id: 'createtarget.specify.area.text',
         })}
-        inputType="number"
-        inputName="radius"
-        inputValue={radius}
-        inputOnChange={handleChange}
-        error={isSubmitted && !radius}
-        errorMsg={intl.formatMessage({
-          id: 'createtarget.missing.radius.text',
-        })}
+      </p>
+      <NumberFormat
+        thousandSeparator={true}
```

Comment:
Isn't it the default value?

Reply:
Not shure, let me check

Reply:
By default there is no separator. So I'll leave it this way